tests/integration/ha/test_backups.py

```diff
@@ -114,44 +119,201 @@ async def c_writes_runner(ops_test: OpsTest, c_writes: ContinuousWrites):
     """Starts continuous write operations and clears writes at the end of the test."""
     await c_writes.start()
     yield
+
+    reachable_ip = random.choice(await get_reachable_unit_ips(ops_test))
+    await http_request(ops_test, "GET", f"https://{reachable_ip}:9200/_cat/nodes", json_resp=False)
+    await http_request(
+        ops_test, "GET", f"https://{reachable_ip}:9200/_cat/shards", json_resp=False
+    )
+
     await c_writes.clear()
     logger.info("\n\n\n\nThe writes have been cleared.\n\n\n\n")
 
 
+@pytest.fixture(scope="session")
+def microceph():
+    """Starts microceph radosgw."""
+    import subprocess
+
+    if "microceph" not in subprocess.check_output(["sudo", "snap", "list"]).decode():
+        import os
+
+        import requests
+
+        uceph = "/tmp/microceph.sh"
+
+        with open(uceph, "w") as f:
+            resp = requests.get(
+                "https://raw.githubusercontent.com/canonical/microceph-action/main/microceph.sh"
```

Comment:
please download this file and add it to the tests resources if needed

Reply:
@Mehdi-Bendriss this is temporary and will go away with: https://github.com/canonical/opensearch-operator/pull/159

Reply:
As discussed: I will add a TODO on each of the lines for microceph